Source/Factories/JSONFactory.swift

```diff
@@ -35,129 +35,125 @@ fileprivate enum RuleKey {
     static let not = "NOT"
 }
 
-/// A factory that wraps `Component` builder closures (ViewFactoryBuilder, `WrapperFactoryBuilder`,
-/// `ClusterFactoryBuilder` & `RuleFactoryBuilder`) and uses them to produce `Component`s
+/// A factory that wraps `Component` builder closures (`SingleBuilder`, `WrapperBuilder`,
+/// `ClusterBuilder` & `RuleBuilder`) and uses them to produce `Component`s
 public final class JSONFactory {
     
-    /// A factory closure to build a view `Component`
-    public typealias ViewFactoryBuilder = (ComponentMeta?) -> Component
+    /// A closure to build a single `Component`
+    public typealias SingleBuilder = (ComponentMeta?) -> Component
```

Comment:
Absolutely; if we name them `XXComponentBuilder`, the purpose and return type would be obvious, and the naming scheme would be more symmetrical w.r.t. the `XXViewBuilder` naming. I was just hesitant to blow up the naming of these, but I'm happy to change this.